composer.json

```
@@ -6,18 +6,29 @@
 		"classmap-authoritative": true,
 		"platform": {
 			"php": "8.0"
+		},
+		"allow-plugins": {
+			"bamarni/composer-bin-plugin": true
 		}
 	},
 	"scripts": {
 		"cs:fix": "php-cs-fixer fix",
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
 		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
-		"test:unit": "vendor/bin/phpunit -c tests/phpunit.xml"
-
+		"test:unit": "vendor/bin/phpunit -c tests/phpunit.xml",
+		"post-install-cmd": [
+			"@composer bin all install --ansi",
+			"composer dump-autoload"
+		],
+		"post-update-cmd": [
+			"@composer bin all update --ansi",
+			"composer dump-autoload"
+		]
+	},
+	"require": {
+		"bamarni/composer-bin-plugin": "^1.8"
 	},
 	"require-dev": {
-		"christophwurst/nextcloud_testing": "^0.12.4",
-		"nextcloud/coding-standard": "^1.0.0",
-		"phpunit/phpunit": "^9.5"
+		"nextcloud/ocp": "^25.0"
```

This app is branched per major version of nextcloud.
So on master it should be:
```suggestion
		"nextcloud/ocp": "dev-master"
```